wds-hero-widget.php

```diff
@@ -229,7 +229,7 @@ function wds_hero( $args = array(), $content = false ) {
 		ob_start();
 		?>
 
-		<div class="hero <?php echo esc_attr( $args['type'] ); ?>-hero <?php echo esc_attr( $args['class'] ); ?>">
+		<div id="id-<?php echo esc_attr( $args['class'] ); ?>" class="hero <?php echo esc_attr( $args['type'] ); ?>-hero <?php echo esc_attr( $args['class'] ); ?>">
```

Comment:
Is there any way we can add an arg called `id` and use that? A little more work, but probably the *right* thing to do rather than use the class over again? What do you think? You could even use `class` as a fallback if no `id` is supplied. I also don't think we need to prefix it with `id-`.